jekyll/_docs/introduction.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: classic-docs
+title: Introduction to Continuous Integration and CircleCI
+short-title: Introduction to CircleCI
+categories: [getting-started]
+description: "An introduction to Continuous Integration, Continuous Deployment, and how CircleCI can help."
+---
+
+Typically CircleCI is used to do [Continuous Integration][wiki-ci] and Continuous Deployment. 
```

Comment:
"CircleCI acts as a platform for both [Continuous Integration][wiki-ci] and Continuous Deployment."